Solutions/Day1/01.c

```diff
@@ -1 +1,44 @@
+#include <stdio.h>
 
+int main()
+{
+    char words[50];
+
+    // Question 1
+    printf("Hello! What's your name? \n");
+    scanf("%s",words);
+    // Answer 1
+    // My name is Saurabh.
+
+    printf("Welcome, %s!\n", words);
+
+    // Question 2
+    printf("Why do you want to learn C? \n");
+    scanf(" %s", words);
```

Comment:
%s will only take one word as input what if you give whole statement as input. Correct the code.